templates/mailservice/mailservice-secret.yaml

```diff
@@ -10,6 +10,6 @@ data:
   SPRING_MAIL_PASSWORD: {{ .Values.springMailPassword | b64enc | quote }}
   MAIL_EXCHANGE_USER: {{ .Values.mailExchangeUser | b64enc | quote }}
   MAIL_EXCHANGE_PASSWORD: {{ .Values.mailExchangePassword | b64enc | quote }}
-  WEBLATE_API_URL: {{ .Values.weblateApiUrl | b64enc | quote }}
-  WEBLATE_API_KEY: {{ .Values.weblateApiKey | b64enc | quote }}
+  WEBLATE_API_URL: {{ .Values.weblate.apiUrl | b64enc | quote }}
```

Comment:
`weblate.api.url` and `weblate.api.key`
would be good because api has multiple keys.